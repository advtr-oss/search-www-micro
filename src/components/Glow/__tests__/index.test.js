/**
 * @jest-environment jsdom
 */

import React from 'react'
import { render } from '@testing-library/react'

import { Glow } from '../index'

describe('Dropdown', function () {
  it('should match snapshot', function () {
    const { container } = render(
      <Glow />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
