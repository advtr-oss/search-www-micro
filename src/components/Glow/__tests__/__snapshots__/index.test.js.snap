// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown should match snapshot 1`] = `
.c0 {
  -webkit-animation: bQashm 1.5s ease-in-out infinite;
  animation: bQashm 1.5s ease-in-out infinite;
  color: transparent;
  cursor: progress;
  display: block;
  width: 100%;
  height: 100%;
}

<span
  class="c0"
/>
`;
